bika/lims/catalog.py

```diff
@@ -62,6 +62,9 @@
             'getPrinted': 'FieldIndex',
             'getClientSampleID': 'FieldIndex',
             'getSampleID': 'FieldIndex',
+            # allowedRolesAndUsers is obligatory if we are going to run
+            # advancedqueries in this catalog.
+            'allowedRolesAndUsers': 'KeywordIndex',
```

Comment:
Yes, but iit is better to do so after accepting the analysis_catalog pull request, because that pull request contains some catalog refactoring.